root/src/mappings/registry.ts

```diff
@@ -0,0 +1,21 @@
+import { TokenMapped } from '../../generated/Registry/Registry'
+import { TokenMapping } from '../../generated/schema'
+
+export function handlePlasmaTokenMapped(event: TokenMapped): void {
+  let id = 'token-mapping-' + event.params.rootToken.toHexString()
```

Comment:
What happens when we have token mapped on both plasma and pos?

We should add test in id:

```suggestion
  let id = 'plasma-token-mapping-' + event.params.rootToken.toHexString()
```

Reply:
Fixed issue, using different `id` for Plasma & PoS root tokens.

Reply:
Thanks for pointing it out @jdkanani 🙌